docs/build_a_connector.rst

```diff
@@ -16,11 +16,19 @@ The first thing you’ll need to do is create a new folder for your connector.
 
 Inside the folder, create two files.  The first should be named __init__.py and should be empty.  The second will have the same name as your folder - this is the file which will have your connector’s code.  For example, in the airtable folder this file is called airtable.py and in the hustle folder it’s called hustle.py.
 
-The directory should look like this::
+The directory should look like this:
 
+.. code-block:: python
     yourconnectorname/
```

Comment:
You'll need to add a blank line between the `.. code-block:: python` directive and the thing we want to display, otherwise it won't show up. I just learned that fact today!